eclair-core/src/main/scala/fr/acinq/eclair/wire/LightningMessageCodecs.scala

```diff
@@ -120,7 +128,8 @@ object LightningMessageCodecs {
   val fundingSignedCodec: Codec[FundingSigned] = (
     ("channelId" | bytes32) ::
       ("signature" | bytes64) ::
-      ("channelData" | channeldataoptional)).as[FundingSigned]
+      ("channelData" | channeldataoptional) ::
+      ("customRemoteSigs" | customremotesigs)).as[FundingSigned]
```

Comment:
I did that on the branch that receives those messages.

Reply:
But shouldn't we:

- have a unit test that writes these messages here
- have a unit test that reads them on the LSP side, with the same hard-coded hex?

That would ensure we're completely consistent, because the part where we read it probably doesn't use the magic but rather a tlv?

Reply:
That's what I did (serialize on _this_ branch, deserialize on _that_ branch), but it's not enough: a test is still required here for non-reg.

Reply:
I verified the test cases, the hex does look like what we expect (and the end should correctly parse as a tlv stream) :+1: 
Were you able to test them against the server code as well?